airbyte-ci/connectors/pipelines/pipelines/cli/dagger_run.py

```diff
@@ -72,6 +72,10 @@ def get_dagger_cli_version(dagger_path: Optional[str]) -> Optional[str]:
 
 
 def check_dagger_cli_install() -> str:
```

Comment:
nit: shall we rename this to `get_dagger_path` that would wrap subfunctions: 
* Checking if the current dagger cli version is the correct one
* Installing it if needed
* Returning the current  dagger cli path